UnityProject/Assets/Scripts/HealthV2/Living/LivingHealthMasterBase.cs

```diff
@@ -829,7 +913,7 @@ public float GetTotalBlood()
 		/// </summary>
 		public float GetSpareBlood()
```

Comment:
I would like to see this kind of methods out of ``LivingHealthMasterBase`` in the future. So anyone who needs the spare blood would instead do ``livingHealthMasterBase.ActiveSystems.OfType<CirculatorySystem>()?.GetSpareBlood()`` or something like that

Reply:
problems is called quite a lot, and it's quite a long Piece of code

Reply:
good thing IDE's exist and can auto complete your stuff

Reply:
is messy though